app/locales/eula/en.html

```diff
@@ -98,7 +98,7 @@
       <span class="c8"
         ><a
           class="c11"
-          href="https://www.google.com/url?q=https://github.com/tripleblindmarket/safe-places&amp;sa=D&amp;ust=1587722145899000"
```

Comment:
@tstirrat  Any idea why we were using a redirect link here? Assuming that it's fine to just use the direct link like @pauladam316 is doing here?

Reply:
It was a save from Google doc to html. I believe that was added by Google and we don't need or want it.